_posts/2022-06-13-is-my-ai-discriminatory/is-my-ai-discriminatory.Rmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Is my AI Discriminatory?"
+description: |
+  A discussion about bias in healthcare AI, and building models with fairness and ethics in mind [5 min read]
+author:
+  - name: Meggie Debnath
+    affiliation: DSAA, Unity Health Toronto
+    affiliation_url: https://chartdatascience.ca
+date: 2022-06-13
+output:
+  distill::distill_article:
+    toc: true
+categories:
+  - post-journal-club
+  - bias-fairness-ethics
+draft: true
+preview: ai-biases.jpg
+references: refer.bib
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+Artificial intelligence (AI) and machine learning (ML), in particular, are undoubtedly shaping how decisions are made across several industries. From identifying fraudulent bank transactions to listing the shows and movies we’re most likely to enjoy, AI is deeply embedded within our everyday lives. Oftentimes the results or outputs of these decisions are relatively harmless. However, increasingly, machine learning models are being trained to operate in sensitive domains such as diagnosing diseases or making hiring decisions. While these models have the ability to transform and improve lives, sometimes the decisions made or informed by AI can have far-reaching consequences.
+
+As a part of our team’s bi-weekly journal clubs, we talked about sources of bias for AI models, the potential consequences and harms they can create, and what we can do as data scientists within the healthcare space.
```

Comment:
@derekbeaton Are there plans to make the journal club Github repo public? If so, we could add a link to the repo here.